### PR TITLE
Add documentation for the scalar types

### PR DESCRIPTION
**Pull request by gjtorikian**
Pretty much what the title says.

### CODE REVIEW

File: lib/graphql/boolean_type.rb [hunk 1]
```diff
@@ -3,6 +3,7 @@
   ALLOWED_INPUTS = [true, false]
 
   name "Boolean"
+  description" The `Boolean` scalar type represents `true` or `false`."
```
Comment (rmosolgo) [inline lib/graphql/boolean_type.rb]: oops, leading whitespace!

File: lib/graphql/float_type.rb [hunk 2]
```diff
@@ -1,5 +1,7 @@
 GraphQL::FLOAT_TYPE = GraphQL::ScalarType.define do
   name "Float"
+  description "The `Float` scalar type represents signed double-precision fractional values as specified by [IEEE 754](http://en.wikipedia.org/wiki/IEEE_floating_point)."
```
Comment (rmosolgo) [inline lib/graphql/float_type.rb]: Do these need to start with `"The {x} scalar type represents..."` Could this one just be 

```ruby 

```ruby
description "signed, double-precision fractional value, as specified by [IEEE 754](http://en.wikipedia.org/wiki/IEEE_floating_point)"
```

?

Comment (gjtorikian) [inline lib/graphql/float_type.rb]: Yeah, I like that. I'll make the change across the types. 👍

**Comment by rmosolgo** [comment]
cool, thanks!

**Pull request merged**